CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
+
+Web3
+
+- Added Solana Swap integration into existing swap UI using Jupiter as the DEX aggregator ([#6847](https://github.com/brave/brave-ios/issues/6847))
+- Added message for Submitted/Completed state change after a transaction is submitted. ([#6960](https://github.com/brave/brave-ios/issues/6960))
+- Added UI to display ‘signed’ status for Ethereum transactions ([#6606](https://github.com/brave/brave-ios/issues/6606))
```

Comment:
‘signed’ -> "Signed" 

Use double quotes as the single quotes can get messy when published in brave.com/latests